src/Amalgamation.sln

```diff
@@ -1,15 +1,14 @@
 ﻿
 Microsoft Visual Studio Solution File, Format Version 12.00
-# Visual Studio Version 16
-VisualStudioVersion = 16.0.29209.62
+# Visual Studio Version 17
+VisualStudioVersion = 17.7.34202.233
 MinimumVisualStudioVersion = 10.0.40219.1
 Project("{9A19103F-16F7-4668-BE54-9A1E7A4F7556}") = "ElastiBuild", "build\ElastiBuild.csproj", "{CDD2CCC6-3C0C-47CC-A7E5-93FC4BF2664A}"
 EndProject
 Project("{2150E333-8FDC-42A3-9474-1A3956D46DE8}") = "script", "script", "{6E880F89-0C6E-4BBC-88F6-5DF297901196}"
 	ProjectSection(SolutionItems) = preProject
 		..\build.bat = ..\build.bat
 		config\config.yaml = config\config.yaml
-		..\global.json = ..\global.json
```

Comment:
The Buildkite job (via the script `.\buildkite\scripts\build.ps1`) [failed](https://buildkite.com/elastic/elastic-stack-installers/builds/2715#018b6c90-3ba6-4325-8977-bb17e58d9c32/45) because it was unable to find the `global.json` file. Is there a reason why it got removed here?

The executed command is:

```
echo "~~~ Installing dotnet-sdk"
& "./tools/dotnet-install.ps1" -NoPath -JSonFile global.json -Architecture "x64" -InstallDir c:/dotnet-sdk
```

Reply:
.netcore 3.1 is EOL, so i thought it's a good opportunity to upgrade. What worked for me (locally) is to just remove the file and yes the .NET version mentioned in the project properties.

Should we bring back the file and try stating a different .NET version?

Reply:
Gotcha. Yes I think we should bring it back. I looked at the possibilities in https://dotnet.microsoft.com/en-us/download/visual-studio-sdks and maybe we can start conservative and try the .NET 6 which is LTS. e.g. I tried this `global.json`:

```json
{
    "sdk": {
      "version": "6.0.319"
    }
  }
```

with this, at least the install step proceeded. Since I tried it on a personal Windows workstation, then I had issues with Vault, but at least we can after that follow up the situation using the actual Buildkite Windows workers.